src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/PrisonConfigService.kt

```diff
@@ -39,4 +52,28 @@ class PrisonConfigService(
   fun getSupportedPrisons(): List<String> {
     return prisonRepository.getSupportedPrisons()
   }
+
+  @Transactional
+  fun createPrison(prisonDto: PrisonDto): PrisonDto {
+    val newPrison = Prison(prisonDto.code, prisonDto.active)
```

Comment:
shall we add a blank validate method that we can populate with rules later to create / activate / deactivate a prison? we could leave it till later as well

Reply:
feature for later on I think! could put a check here to see if it exist mind and if it does it throws an exception